static/css/components/donate.less

```diff
@@ -47,7 +34,19 @@ h2.donation__override {
       transform: translateY(-50%);
     }
   }
-  note {
+  &note {
     color: @light-grey;
   }
-}
\ No newline at end of file
+}
+
+div.donateModal {
```

Comment:
we can probably drop the div. here. This doesn't look it needs to be so specific. This also has me wondering why this is not using a BEM class like the others. e.g. donate__model